experimental/generator-dotnet-yeoman/generators/app/templates/dotnet/functions/ActivitySerializationSettings.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Microsoft.Rest.Serialization;
+using Newtonsoft.Json;
+
+namespace <%= botName %>.FunctionHost
```

Comment:
Do we need to keep the '.FunctionHost' suffix? I would drop that part.

Reply:
This applies as a general comment throughout for the Functions host template C# files.

Reply:
Consistent with WebHost